WRIO-InternetOS/js/components/CreateDomCenter.js

```diff
@@ -1,3 +1,10 @@
+'use strict';
```

Comment:
I do not commit later

2015-12-08 15:51 GMT+03:00 Michael Bilenko <notifications@github.com>:

> In WRIO-InternetOS/js/components/CreateDomCenter.js
> <https://github.com/webRunes/WRIO-InternetOS/pull/405#discussion_r46948642>
> :
>
> > @@ -1,3 +1,10 @@
> > +'use strict';
>
> @Xurma2 <https://github.com/Xurma2> why then i still see this line?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webRunes/WRIO-InternetOS/pull/405/files#r46948642>.
>